BLSQ/iaso
# Registry children list contains selected org unit

While visiting new registry page on tab list, it was also containing the selected org unit.


## Self proofreading checklist


## Changes

Filtering data result

## How to test
Go to to registry page for a specific org unit on list tab, check that the list does not contain the selected org unit.
